include/enrico/surrogate_heat_driver.h

```diff
@@ -3,18 +3,17 @@
 #ifndef ENRICO_SURROGATE_HEAT_DRIVER_H
 #define ENRICO_SURROGATE_HEAT_DRIVER_H
 
-#include "driver.h"
-
+#include "enrico/heat_fluids_driver.h"
 #include "pugixml.hpp"
+#include "mpi.h"
 #include "xtensor/xtensor.hpp"
-#include <gsl/gsl>
-#include <mpi.h>
 
 #include <cstddef>
+#include <gsl/gsl>
```

Comment:
This is not in the C++ standard library -- it belongs in the third-party library section.

Reply:
Another question.  The core guidelines seem to always list headers with `< >` instead of `" "`, even for headers that aren't part of the standard library.  Should I follow that convention?

Reply:
`< >` means "search for this file in the standard locations" whereas `" "` means "search for this file in the standard locations _and_ the present directory". The only case where `" "` is actually needed is if the file being included is truly in a relative location (that is not on the include path). For example, in "heat_fluids_driver.h", we have:
```C++
#include "driver.h"
```
If this were
```C++
#include <enrico/driver.h>
```
it would work because `include/enrico` is on the include path. With `" "` however we have to rely on it being relative.

Reply:
From a practical standpoint, I usually use `< >` to indicate "external to this project" and `" "` to indicate "internal to this project".